ts/packages/node/src/stores/TomlFile.ts

```diff
@@ -0,0 +1,80 @@
+import { ConfigStore, ConfigStoreQuery, Config, Convert } from '@configu/ts';
+import _ from 'lodash';
+import { promises as fs } from 'fs';
+import toml from '@iarna/toml';
```

Comment:
This requires the addition of the package in the dependencies in `package.json`.